pkg/systemd/terminal.jsx

```diff
@@ -186,6 +268,22 @@ const _ = cockpit.gettext;
                             </ToolbarContent>
                         </Toolbar>
                     </div>
+                    <div className="ct-terminal-dir-alert">
+                        {this.state.pathError && <Alert isInline variant='danger'
```

Comment:
This isn't really a "danger", there's no potential data loss here. Please change to "warning".